aws/services/cw/policy.ftl

```diff
@@ -21,7 +21,24 @@
         [
             getPolicyStatement(
                 [
-                    "cloudwatch:PutMetricData",
+                    "cloudwatch:PutMetricData"
+                ],
+                "*",
+                "",
+                logGroupName?has_content?then(
+                    {
+                        "StringEquals": {
+                            "cloudwatch:namespace": 
+                                "CWAgent"+logGroupName?keep_before_last("/")?keep_before_last("/")
+                        }
```

Comment:
This feels quite confusing and I wouldn't know the name straight off, since its used for alerts it needs to be easy to understand what the value will be 

- I think this would be better as its own macro that returns a separate policy specific for metric data pushing. And that has a namespace value. That way its formatted outside of the policy statement and can be included as part of the ec2 instance state potentially 
- We could also leave out creating a dedicated namespace for each component and rely on the dimensions provided for each ec2 instance. This would align with how metrics are collected. Then the permission applied would always been to the CWAgent namespace for these metrics and the dimensions can be filtered. That still limits the potential for overriding metrics but makes it easier to understand and aligns with how metrics are stored across the board

Reply:
Addressed in 1bd6d75